robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
I mean, from user perspective - for one fetching helper he should not navigate anywhere and pass a list of desired properties, for another helper he should not forget to search and open the host and pass properties one by one. We should probably stick with something single (but yeah, separate helper which won't press 'edit' button and doesn't switch tabs is needed here).